db/migrate/20191206082828_add_type_to_pages.rb

```diff
@@ -0,0 +1,10 @@
+class AddTypeToPages < ActiveRecord::Migration[6.0]
```

Comment:
Add Page class definition

```
Page = Class.new(ActiveRecord::Base)
```

as a good practive when migrating data and using AR models.

Reply:
Thanks, will add!